src/Testing/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -101,6 +103,17 @@ protected function setUp(): void
             $rectorNodeTraverser = $rectorConfig->make(RectorNodeTraverser::class);
             $rectorNodeTraverser->refreshPhpRectors($rectors);
 
+            $postRectorsGenerator = $rectorConfig->tagged(PostRectorInterface::class);
```

Comment:
When running the tests we need to inject any additional post rector defined in the test config into the post file processor